src/Microsoft.Performance.SDK.Tests/TableDescriptorFactoryTests.cs

```diff
@@ -280,5 +308,31 @@ public bool TryCreateTable(ITableBuilder tableBuilder)
                 "Description",
                 "Category");
         }
+
+        [Table]
+        public sealed class ExtensionTable
+        {
+            public static TableDescriptor TableDescriptor { get; } = new TableDescriptor(
+                Guid.Parse("{DEC27D7E-D77F-4D64-B055-2A5AAE44E2F9}"),
+                "Name",
+                "Description",
+                "Category",
+                requiredDataCookers: new DataCookerPath[] { DataCookerPath.ForSource("blah", "blahblah") });
```

Comment:
I have a feeling we might get a email later b/c of "blah" 😄

Reply:
huh?

Reply:
Might be good to use a different word then "blah"